app/views/devise/registrations/new.html.erb

```diff
@@ -1,29 +1,47 @@
-<h2>Sign up</h2>
+<div class="col-xs-1 col-sm-2 col-md-3 col-lg-4 sign-up-column-position">
```

Comment:
Try to not use this, and still make the thing centered. Using 'offset' for columns in bootstrap